server/routes/approvingLicences/handlers/approvalCases.ts

```diff
@@ -26,7 +26,12 @@ export default class ApprovalCaseRoutes {
           name: convertToTitleCase(`${c.nomisRecord.firstName} ${c.nomisRecord.lastName}`.trim()),
           prisonerNumber: c.nomisRecord.prisonerNumber,
           probationPractitioner: c.probationPractitioner,
-          releaseDate: moment(c.nomisRecord.releaseDate || c.nomisRecord.conditionalReleaseDate).format('DD MMM YYYY'),
+          releaseDate: moment(
```

Comment:
I would break out this logic into a function. The CRD must always be set, but it could be possible to have both other values. Something like a simple switch statement with default of CRD would work. It might be worth trying https://date-fns.org/ as part of the moment replacement task. But it's not essential.